pkg/serverinit/genconfig.go

```diff
@@ -755,7 +759,8 @@ func (b *lowBuilder) addGoogleCloudStorageConfig(v string) error {
 	if isReplica {
 		gsPrefix := "/sto-googlecloudstorage/"
 		b.addPrefix(gsPrefix, "storage-googlecloudstorage", args{
-			"bucket": bucket,
+			"bucket":     bucket,
+			"rate_limit": rate,
```

Comment:
I think everything else in Perkeep (+ JSON in general?) uses fooBar case, not underscores?

Reply:
Solved this (and the obviousness of units) by renaming the config item to `qps`.